src/components/side-navigation/SideNavigation.tsx

```diff
@@ -4,28 +4,36 @@ import NavigationHeader from './NavigationHeader';
 import NavigationItem from './NavigationItem';
 import { SideNavigationContext } from '../../store/SideNavigationContext';
 import { termsData } from '~pages/term/termsData';
+import Overlay from '~components/UI/Buttons/overlay/Overlay';
 /**
  * The side navigation bar of the website.
  */
 export default function SideNavigation() {
   const sideNavContext = useContext(SideNavigationContext);
   const isOpen = sideNavContext.isSideNavigationOpen;
+
+  const closeSideNavigation = () => {
+    sideNavContext.closeSideNavigation();
+  };
   if (!isOpen) {
     return null;
   }
   return (
```

Comment:
Add new line seperation.

```suggestion
  };

  if (!isOpen) {
    return null;
  }
  
  return (
```